modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/App.es.js

```diff
@@ -39,6 +39,10 @@ export default (props) => {
 		path = props.i18nPath + path;
 	}
 
+	if (path && location.pathname.endsWith(path)) {
```

Comment:
Fiuuuu

Reply:
It is quite ugly... I wanted to suggest that, if we need to match with the page friendlyUrl, turn this configuration parameter to a checkbox and take the location.pathname directly